### ISSUE TITLE
Semaphore Task hangs when some of ansible tasks have massive output

### ISSUE THREAD

**Issue by strangeman** [ansible-semaphore/semaphore#393]
In some cases, I want to see large file diffs (when applies major config changes) or large verbose outputs of tasks (`unzip` task for big archive, for example). Semaphore just hangs in this cases, it not finishing current task and not launch next tasks and jobs. 

```
04:42:23: RUNNING HANDLER [unzip task with verbose log, sometimes about 500Kb of text output] *****************************
04:42:23: Friday 07 July 2017  21:42:23 +0300 (0:00:00.838)       0:00:55.968 *********** 
10:47:06: Running playbook failed: signal: terminated
```

I set 
```
      proxy_buffering off;
      proxy_request_buffering off;
      proxy_read_timeout 300s;

```
on Nginx side, but it won't helped, so I think this is Semaphore issue.

CLI Ansible working as expected.

**Comment by matejkramny** [comment]
it won't be with nginx as the log says the playbook failed. Needs looking into, maybe it is having issues forwarding the log to the database.

**Comment by strangeman** [comment]
Playbook failed because I killed `ansible-playbook` process after 4 hours of waiting. If I do not kill it - it hangs literally forever.

**Comment by strangeman** [comment]
After adding an error handling I've got this error:

```
panic: bufio.Scanner: token too long

goroutine 688 [running]:
github.com/ansible-semaphore/semaphore/api/tasks.(*task).logPipe(0xc4200fd180, 0xc420267a00)
	/mnt/src/sources/go/src/github.com/ansible-semaphore/semaphore/api/tasks/logging.go:73 +0x107
created by github.com/ansible-semaphore/semaphore/api/tasks.(*task).logCmd
	/mnt/src/sources/go/src/github.com/ansible-semaphore/semaphore/api/tasks/logging.go:82 +0x135
```

Looks like `bufio.Scanner` can't properly work with large strings. I'll try to implement another solution.

**Comment by matejkramny** [comment]
@strangeman how can we reproduce the error?

**Comment by strangeman** [comment]
First, you need to enable debug for `ansible-playbook` (`-v` key)

Then, you need to execute task like:
```
- name: Put 1M text buffer with one looooooooooong line
  shell: tr -dc A-Za-z0-9 </dev/urandom | head -c 1048576
```
Semaphore Task will hang.

Also, you can easily reproduce https://github.com/ansible-semaphore/semaphore/issues/132 by echoing  many lines:

```
    - local_action: debug msg="Hi! I kill your Semaphore, LOL"
      with_sequence:
        - count=10000
```

**Comment by sanders54** [comment]
I was able to reproduce this. It also killed the webserver...

**Comment by matejkramny** [comment]
Thanks, will investigate

**Comment by twhiston** [comment]
will be fixed when #395 is merged
